draft-ietf-quic-multipath.md

```diff
@@ -1246,9 +1249,6 @@ also needs to be considered in the context of the Path Identifier field.
 A MAX_PATH_ID frame (type=0x15228c0c) informs the peer of the maximum path identifier
 it is permitted to use.
 
-When there are not enough unused path identifiers, endpoints SHOULD
-send MAX_PATH_ID frame to inform the peer that new path identifiers are available.
```

Comment:
This is removed here because it is already specified normatively in section 3.5. [Allocating, Consuming, and Retiring Connection IDs](https://quicwg.org/multipath/draft-ietf-quic-multipath.html#name-allocating-consuming-and-re)